test/spec/services/membership/membershipSpec.js

```diff
@@ -169,19 +186,55 @@ describe('MembershipService', function() {
   });
 
   describe('#getSubjectKinds', function() {
-    // No test needed, this returns static data
+    it('should return the kind list in the preferred ourder', function() {
+      var kinds = MembershipService.getSubjectKinds();
```

Comment:
If this getSubjectKinds is returning a map, there is no guarantee of iteration order. If you are expecting stable order that method probably needs to change.  If its returning an array and not a map I'm not sure why the next line is needed.